packages/kmod-6.12-amdgpu/kmod-6.12-amdgpu.spec

```
@@ -0,0 +1,74 @@
+%global kernel_sources %{_cross_usrsrc}/kernels/6.12
+%define _kernel_version %(cat %{kernel_sources}/include/config/kernel.release)
+%global _cross_kmoddir %{_cross_libdir}/modules/%{_kernel_version}
```

define `amdgpu_kmoddir` to cut down on repetition of the install path:
```suggestion
%global _cross_kmoddir %{_cross_libdir}/modules/%{_kernel_version}
%global amdgpu_kmoddir %{_cross_kmoddir}/kernel/drivers/extra/gpu/drm/amd/amdgpu
```